tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -104,6 +104,34 @@ Run Docker Checks
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${output}  Exited (0)
 
+    # check the display name and datastore folder name of an existing container
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'VC'  Run And Return Rc And Output  govc vm.info %{VCH-NAME}/*-%{ID1}
+    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should Be Equal As Integers  ${rc}  0
+    Run Keyword If  '%{HOST_TYPE}' == 'VC'  Should contain  ${output}  vch-restart-tes-%{ID1}
+    ${rc}  ${output}=  Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Run And Return Rc And Output  govc vm.info *-%{ID1}
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Be Equal As Integers  ${rc}  0
+    Run Keyword If  '%{HOST_TYPE}' == 'ESXi'  Should Contain  ${output}  vch-restart-tes-%{ID1}
+    ${rc}  ${output}=  Run And Return Rc And Output  govc datastore.ls | grep vch-restart-tes-%{ID1}
```

Comment:
@mhagen-vmware For an existing container before VCH upgrade, the pretty name in the display name would be truncated if it is longer than 15 chars. So `vch-restart-test1` becomes `vch-restart-tes` and the display name becomes `vch-restart-tes-id`